products/health-care/beneficiary-travel/engineering/data-flow-sequence-diagram.md

```diff
@@ -0,0 +1,33 @@
+The basic sequence for most BTSSS is straightforward:
+
+```mermaid
+sequenceDiagram
+autonumber
+actor veteran as Veteran
+box VA.gov
+participant vets-website
+participant vets-api
+end
+participant travel-pay-api as Travel Pay API
+participant dynamics-db as Dynamics DB
+
+veteran->>vets-website: Navigate to travel pay
+
+activate vets-website
+
+vets-website->>vets-api: vets-api endpoints (/claims, /appointments, etc.)
+vets-api->>travel-pay-api: travel-pay endpoints (/claims, /appointments, etc.)
+travel-pay-api->dynamics-db: fetch data via dynamics portal plugins
+
+dynamics-db-->>travel-pay-api: claims and appoinments data
+travel-pay-api-->>vets-api: travel-pay endpoint response(s)
```

Comment:
Thinking about performance - what would adding in Redis caching look like here? I'm not sold on if we need to do that with claim status responses, but might be good to think about for later.

Also, do you think it's necessary to add caching in?

Reply:
Since it's likely that the requests made by the travel pay API will have the biggest performance implications, I think caching from there to start is the most important. I'm not sure the impact caching vets-api responses on top of that will be as critical, but it's worth looking into. 

> what would adding in Redis caching look like here? 

I'm not quite sure what the additional complexity/level of effort doing this in `vets-api` entails

Reply:
Added in c57c7b79a53ae276626d412da51b99afe996ed27